ipfx/nwb_reader.py

```diff
@@ -111,17 +113,17 @@ def get_unit_name(stim_attrs):
     def get_long_unit_name(unit):
         if not unit:
             return "Unknown"
-        if unit.startswith('A'):
+        elif unit.startswith('A') or unit == 'amps':
```

Comment:
is this as specific of a check we need? `startswith` seems possibly ambigious...

Reply:
I specified unit names explicitly as you suggested. thanks.